cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: che-server
```

Comment:
@l0rd Yes, this is required to build the image in CentOS pipeline. It's referenced [here](https://github.com/CentOS/container-index/blob/master/index.d/eclipse.yml#L70).